README.md

```diff
@@ -30,5 +30,270 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r21.12 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.17.0
+
+* Improved [Inferentia support](https://github.com/triton-inference-server/python_backend/tree/r21.12/inferentia#readme)
+  to use Neuron Runtime 2.x and multiple instances.
+
+* Models from MLflow can now be deployed to Triton with the [MLflow 
+  plugin](deploy/mlflow-triton-plugin).
+
+* The preview release of TorchTRT models is now supported. PyTorch models 
+  optimized using TensorRT can now be loaded into Triton in the same way as 
+  other PyTorch models.
+
+* At the end of each Model Analyzer phase, an example command line will be 
+  printed to run the next phase.
+
+## Features
```

Comment:
The main README.md file has been updated. The release branch README should be based on the main branch README at the time the release branch was created. It could be that the main branch changes didn't happen till after the 21.12 branch was created... if so then we will pick them up for 22.01.